app/controllers/application_controller.rb

```diff
@@ -36,8 +36,12 @@ def authenticated?
     current_user.present?
   end
 
+  def save_redirect_path
```

Comment:
I think the path is really the requested path rather than the redirect path. Would `save_requested_path` and `session[:requested_path]` be clearer?

Reply:
Sorry.. just realised I called it `redirect_back_to` in the Trello ticket 🤦